### PR TITLE
Add template for minimal image and update tutorial

### PR DESCRIPTION
**Pull request by vpolikarpov-akvelon**
# Description
Update image generation tutorial and provide template for minimal ubuntu image that may be created in 30 minutes.

## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Review by erik-bershel — APPROVED** [review]

File: helpers/GenerateResourcesAndImage.ps1 [hunk 1]
```diff
@@ -5,6 +5,7 @@ enum ImageType {
     Windows2022 = 2
     Ubuntu2004 = 3
     Ubuntu2204 = 4
+    UbuntuMinimal = 5
```
[the remark this answers is not in the document]
Comment (samvantran) [reply helpers/GenerateResourcesAndImage.ps1:8]: Yup let's start small. As long as we add apt, curl, and whatever other tools are needed to enable users to download software during a workflow run, that would be enough for a minimal base image.

[the remark this answers is not in the document]
Comment (samvantran) [reply helpers/GenerateResourcesAndImage.ps1:8]: Think of it like a hello world image

[the remark this answers is not in the document]
Comment (vpolikarpov-akvelon) [reply helpers/GenerateResourcesAndImage.ps1:8]: Hey @samvantran. I updated image, removed some software. Now the whole process takes 16 minutes, build itself takes less 10 minutes. Is it ok?

[the remark this answers is not in the document]
Comment (samvantran) [reply helpers/GenerateResourcesAndImage.ps1:8]: Excellent!

**Review by mikhailkoliada — APPROVED** [review]

**Pull request merged**